site/using-weave/host-network-integration.md

```diff
@@ -38,6 +38,20 @@ Exposed addresses can also be added to weaveDNS by supplying fully qualified dom
     host2$ weave expose -h exposed.weave.local
     10.2.1.132
 
+###<a name="routing"></a>Routing from another host
+
```

Comment:
Title case for the title.  This means only using capital letters for the principal words. Articles, conjunctions, and prepositions do not get capital letters unless they start the title.